src/class-trending-post-queries.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Trending_Post_Queries class file
+ *
+ * @package wp-type-extensions
+ */
+
+namespace Alley\WP\WP_Curate;
+
+use Alley\WP\Types\Post_Queries;
+use Alley\WP\Types\Post_Query;
+use Alley\WP\Post_Query\WP_Query_Envelope;
+use Alley\WP\WP_Curate\Features\Parsely_Support;
+use Alley\WP\Post_Query\Post_IDs_Query;
+
+/**
+ * Pull trending posts from Parsely.
+ */
+final class Trending_Post_Queries implements Post_Queries {
```

Comment:
If this is specific to Parse.ly, then it should be named as such. If it's intended to be a generic trending posts class, then it should be structured so that it supports multiple providers, of which Parse.ly will be the first (and we could add other providers like GA4 later).

Reply:
@kevinfodness supporting multiple providers was my plan... I kind of figured we would sort out how structure it when we added the second provider.

Reply:
Ah, we're using named arguments instead of positional ones, this works for me!